pkg/tagger/local/fake_tagger.go

```diff
@@ -100,6 +100,18 @@ func (f *FakeTagger) Standard(entity string) ([]string, error) {
 	return f.store.lookupStandard(entity)
 }
 
+// GetEntity returns faked entity corresponding to the specified id and an error
```

Comment:
Is this change expected in this PR? Feel free to keep it if it makes your PR management easier.